<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Petr Koup√Ω</author>
 <class>CaptureDialog</class>
 <widget class="QDialog" name="CaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelRecorder">
         <property name="text">
          <string>Encoder:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxRecorder</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxRecorder"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelWidth">
         <property name="text">
          <string>Width:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxWidth</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxWidth">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelHeight">
         <property name="text">
          <string>Height:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxHeight</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxHeight">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelStart">
         <property name="text">
          <string>Start:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxStart</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxStart">
         <property name="suffix">
          <string>. timestep</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelStop">
         <property name="text">
          <string>Stop:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxStop</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxStop">
         <property name="suffix">
          <string>. timestep</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelInterval">
         <property name="text">
          <string>Interval:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxInterval</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBoxInterval">
         <property name="suffix">
          <string> timesteps</string>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditName</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditName">
         <property name="text">
          <string>captured</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelPath">
         <property name="text">
          <string>Path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPath</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutPath">
         <item>
          <widget class="QLineEdit" name="lineEditPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonPath">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBottom">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCapture">
       <property name="text">
        <string>&amp;Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cance&amp;l</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxRecorder</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>spinBoxHeight</tabstop>
  <tabstop>spinBoxStart</tabstop>
  <tabstop>spinBoxStop</tabstop>
  <tabstop>spinBoxInterval</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditPath</tabstop>
  <tabstop>buttonPath</tabstop>
  <tabstop>buttonCapture</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
